src/applications/simple-forms/20-10207/pages/evidenceALS.js

```diff
@@ -0,0 +1,76 @@
+import environment from 'platform/utilities/environment';
+
+import { titleUI } from 'platform/forms-system/src/js/web-component-patterns/titlePattern';
+import { FileField } from '../components/FileField';
+import AlsViewField from '../components/AlsViewField';
+
+import { ALS_DESCRIPTION } from '../config/constants';
+import { createPayload, parseResponse } from '../helpers';
+
+const uiTitle =
+  'Upload evidence for diagnosis of ALS (amyotrophic lateral sclerosis)';
+const uiDescription =
+  'If you have documents you would like to submit as evidence of ALS, upload them here.';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
```

Comment:
not sure if it makes sense to patternize this or not,  for example abstracting all the stuff that will usually stay the same. but this does seem like a lot of duplicate code. even local patterns could be fine.

Reply:
Good point.  I'll try DRYing in part 5.  Thanks!